Error on `make test` with go1.10.1

Environment:
Minikube version : v0.26.1
OS : Ubuntu 16.10
GOLANG : 1.10.1

Facing following error while running test cases using go1.10.1
$ make test
```
./test.sh
Running go tests...
# k8s.io/minikube/cmd/minikube/cmd
cmd/minikube/cmd/start.go:322: Println arg list ends with redundant newline
FAIL    k8s.io/minikube/cmd/minikube/cmd [build failed]
Makefile:201: recipe for target 'test' failed
make: *** [test] Error 2
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,7 +3,7 @@ os: linux
 sudo: required
 
 go:
-  - 1.9.x
+  - 1.10.x
 go_import_path: k8s.io/minikube
 
 install:

--- a/Makefile
+++ b/Makefile
@@ -22,8 +22,8 @@ INSTALL_SIZE ?= $(shell du out/minikube-windows-amd64.exe | cut -f1)
 BUILDROOT_BRANCH ?= 2018.05
 REGISTRY?=gcr.io/k8s-minikube
 
-HYPERKIT_BUILD_IMAGE 	?= karalabe/xgo-1.8.3
-BUILD_IMAGE 	?= k8s.gcr.io/kube-cross:v1.9.1-1
+HYPERKIT_BUILD_IMAGE 	?= karalabe/xgo-1.10.x
+BUILD_IMAGE 	?= k8s.gcr.io/kube-cross:v1.10.1-1
 ISO_BUILD_IMAGE ?= $(REGISTRY)/buildroot-image
 
 ISO_VERSION ?= v0.28.1

